src/site/components/preview/preview.11tydata.js

```diff
@@ -20,4 +20,7 @@ const examples = [...paths, ...localExamples].map((path) => {
 
 module.exports = {
   examples,
+  visual_regression: {
+    selector: undefined,
```

Comment:
Need to unset this here as it inherits the value from the parent directory otherwise.